### PR TITLE
Update version: OpenJS.NodeJS version 20.6.1

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/119237)

### CODE REVIEW

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]
It shouldn't remove arm64

**Comment by stephengillie** [comment]
Manifest Error: Duplicate installer entry found.

File: manifests/o/OpenJS/NodeJS/20.6.1/OpenJS.NodeJS.installer.yaml [hunk 1]
```diff
@@ -21,16 +21,16 @@ AppsAndFeaturesEntries:
...
-  InstallerUrl: https://nodejs.org/dist/latest/node-v20.6.1-x86.msi
+  InstallerUrl: https://nodejs.org/dist/v20.6.1/node-v20.6.1-x86.msi
   InstallerSha256: CB83586AF83182187E760B7E01AA7C7B2BACB521D60CEEFED3AC6FC62C222449
   ProductCode: '{29172FB0-B69C-49A9-A24F-780E7A79E3D5}'
 - Architecture: x64
-  InstallerUrl: https://nodejs.org/dist/latest/node-v20.6.1-x64.msi
+  InstallerUrl: https://nodejs.org/dist/v20.6.1/node-v20.6.1-x64.msi
+  InstallerSha256: 5C2616DA46728DD1326645C7DB114E78AD87138A258C0724A035269258C23509
+  ProductCode: '{B96683C4-F8EF-4FBF-9C66-5A41AB9322F0}'
+- Architecture: x64
```
Comment (stephengillie) [inline manifests/o/OpenJS/NodeJS/20.6.1/OpenJS.NodeJS.installer.yaml:31]:
```suggestion
- Architecture: arm64
```

File: manifests/o/OpenJS/NodeJS/20.6.1/OpenJS.NodeJS.installer.yaml [hunk 2]
```diff
@@ -21,16 +21,16 @@ AppsAndFeaturesEntries:
...
 - Architecture: x64
-  InstallerUrl: https://nodejs.org/dist/latest/node-v20.6.1-x64.msi
+  InstallerUrl: https://nodejs.org/dist/v20.6.1/node-v20.6.1-x64.msi
   InstallerSha256: 5C2616DA46728DD1326645C7DB114E78AD87138A258C0724A035269258C23509
   ProductCode: '{B96683C4-F8EF-4FBF-9C66-5A41AB9322F0}'
 - Architecture: arm64
-  InstallerUrl: https://nodejs.org/dist/latest/node-v20.6.1-arm64.msi
-  InstallerSha256: 9471BD6DC491E09C31B0F831F5953284B8A6842ED4CCB98F5C62D13E6086C471
-  ProductCode: '{B81B8D53-19F5-4195-BF91-53710B91C325}'
+  InstallerUrl: https://nodejs.org/dist/v20.6.1/node-v20.6.1-x64.msi
```
Comment (stephengillie) [inline manifests/o/OpenJS/NodeJS/20.6.1/OpenJS.NodeJS.installer.yaml:32]:
```suggestion
  InstallerUrl: https://nodejs.org/dist/v20.6.1/node-v20.6.1-arm64.msi
```

File: manifests/o/OpenJS/NodeJS/20.6.1/OpenJS.NodeJS.installer.yaml [hunk 3]
```diff
@@ -21,16 +21,16 @@ AppsAndFeaturesEntries:
...
   InstallerSha256: 5C2616DA46728DD1326645C7DB114E78AD87138A258C0724A035269258C23509
   ProductCode: '{B96683C4-F8EF-4FBF-9C66-5A41AB9322F0}'
 - Architecture: arm64
-  InstallerUrl: https://nodejs.org/dist/latest/node-v20.6.1-arm64.msi
-  InstallerSha256: 9471BD6DC491E09C31B0F831F5953284B8A6842ED4CCB98F5C62D13E6086C471
-  ProductCode: '{B81B8D53-19F5-4195-BF91-53710B91C325}'
+  InstallerUrl: https://nodejs.org/dist/v20.6.1/node-v20.6.1-arm64.msi
+  InstallerSha256: 5C2616DA46728DD1326645C7DB114E78AD87138A258C0724A035269258C23509
+  ProductCode: '{B96683C4-F8EF-4FBF-9C66-5A41AB9322F0}'
```
Comment (stephengillie) [inline manifests/o/OpenJS/NodeJS/20.6.1/OpenJS.NodeJS.installer.yaml:34]:
```suggestion
  InstallerSha256: 9471BD6DC491E09C31B0F831F5953284B8A6842ED4CCB98F5C62D13E6086C471
  ProductCode: '{B81B8D53-19F5-4195-BF91-53710B91C325}'
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**